### ISSUE TITLE
[Terminal]: 'Duplicate' button should remain in disabled state until user don't select a profile radio control.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12056]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Activate 'Add a new profile' tab and then navigate 'Duplicate' button.
4. Observe the issue.

**User Experience:**
Users UX will not be good especially Screen Reader as nothing will happen when they activate 'Duplicate' button. 

**Attachments:**
['Duplicate' button should remain in disabled state until user don't select a profile radio control..zip](https://github.com/microsoft/terminal/files/7783152/Duplicate.button.should.remain.in.disabled.state.until.user.don.t.select.a.profile.radio.control.zip)

### Expected Behavior

'Duplicate' button should remain in disabled state until user don't select a profile radio control.

### Actual Behavior

'Duplicate' button is in enabled state even when user don't select a profile radio control.

**Note Issue:** 'Save' and 'Discard' button should remain in disabled state until we don't make any changes.

**Comment by rinkydevi** [comment]
hey @v-rpundir ! I would to contribute in this issue, how can i proceed ?

**Comment by zadjii-msft** [comment]
@rinkydevi Sorry for the confusion here. This was actually already fixed in #12096 (and then we entirely changed this UI again in #12326 such that this was no longer an issue). There's just a bit of a delay between when a11y bugs are fixed and when we can close them out. I'll yank off the help wanted/easy starter tags, since that's confusing. 

@v-rpundir can we confirm that this one is no longer an issue?

**Comment by rinkydevi** [comment]
@zadjii-msft thank u for your response sir!

**Comment by v-rpundir** [comment]
@zadjii-msft Main issue is fixed but Note issue still repro.

**Note Issue:** 'Save' and 'Discard' button should remain in disabled state until we don't make any changes.

**Comment by zadjii-msft** [comment]
@v-rpundir Ah okay. Missed that bit. That's actually quite a bit more complicated than disabling this one button till the user makes a selection. The Terminal doesn't have a good way to check when the user has actually made any changes via the UI. Also, the button will technically write out their settings to the file if they hit save even without changes. (I could have sworn there was a specific issue tracking this laying around but all I could find was a single bulletpoint in the  https://github.com/microsoft/terminal/issues/6800 megathread).

For tracking purposes, what do you think the best strategy is here? 
* file a new issue for a11y tracking of the save/discard bug?
* wholly repurpose this thread to track that bug?

I'd rather not have two separate issues tracked in the same thread, so I just want to clarify a bit. Thanks!

**Comment by v-rpundir** [comment]
@zadjii-msft Closing this bug as Main issue is fixed and logged new separate issue for Note https://github.com/microsoft/terminal/issues/12424
